### ISSUE TITLE
[BUG] On SR you get the error 'Unknown command:"LED_HOTEND_OFF"'

### ISSUE THREAD

**Issue by CaryBab** [Guilouz/KlipperScreen-Flsun-Speeder-Pad#12]
**Describe the bug**

10:42 AM
Unknown command:"LED_HOTEND_OFF"
10:42 AM
echo: Printing canceled!
10:42 AM
LED_HOTEND_OFF
*To Reproduce**

Cancel a print on SR

**Expected behavior**
No Errors

**Logfile**
Attach your [klipperscreen.log](https://klipperscreen.readthedocs.io/en/latest/Troubleshooting/)
[KlipperScreenClipped.log](https://github.com/Guilouz/KlipperScreen-Flsun-Speeder-Pad/files/11860767/KlipperScreenClipped.log)

**Comment by Guilouz** [comment]
This is not supposed to happen if you use my configurations because this macro is present.

However I fixed this error in case of absence of this macro. Please update.

**Comment by CaryBab** [comment]
Thanks Cyril.  I will update as soon as my current print is finished.

I have been working on setting up a Euclid Probe.  Have you done anything
more with yours.  I have a mount and a Dock for the SR that I am working
on.  It mounts on the right front post outside the print bed.  If you are
interested I will send you the STL's when I get a final version.

Cary Babcock

**Comment by Guilouz** [comment]
To tell the truth, I abandoned the idea of Euclid Probe.

**Comment by CaryBab** [comment]
Once I found a place I could put the docking port, I decided to give it a
shot.  My bed mesh has never been so good.  It's range now is .167mm.
Prior to your new speederpad klipper firmware and the Euclid probe it was
in the 8mm range.  In your previous comment about using your configs, I do
use them.  I just updated the macro.cfg that you updated a few days ago.
But thanks for fixing it so fast.  I looked for it, but I could not find it
in any of the config files.

Cary

**Comment by CaryBab** [comment]
Cyril, I didn't want to open a problem just in case it is me.

When I first installed your klipper speederpad sw, I was able to do a
zoffset from the panel and it saved it fine.

I have done several updates since then, and when I tried doing a zoffset
today, it does not save it like it did before in the variables.cfg file.

Could something have changed in one of the updates in the last month, or is
it something I'm doing wrong.

I even deleted the printer.cfg and put in a brand new one from your config
files.

Sorry to bother you.

Cary Babcock
